pyproject.toml

```diff
@@ -4,15 +4,16 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "ansible-variables"
-version = "0.8.1"
+version = "0.8.2"
```

Comment:
please don't change the version, this will be done automatically during releasing

Reply:
Sorry, was not aware of that. I dropped the commit that changes the version now.